controller/src/conf/logback.xml

```diff
@@ -18,16 +18,14 @@
 
     <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>controller-server.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
```

Comment:
This change will not be used in our docker images. The configuration under dist/conf will be used..

Reply:
fixed logback.xml in dist/conf

Reply:
Confirmed after logging into docker container  that the updated logback .xml was used.